### PR TITLE
Open IRC blogs in new tabs

### LINKED ISSUE #1152: Add (target =_blank) feature to IRC page "VIEW BLOG" buttons
**Is your feature request related to a problem? Please describe.**
The view blog button on the IRC page redirects the user to another tab. We have to make them open the blogs in separate tabs to keep the user on our website.

**Describe the solution you'd like**
Add suitably attributes to the HTML code to give the functionality of opening the blogs in a new tab

**Describe alternatives you've considered**
Recommended target = _blank

**Additional context**
Screenshot (704)

### PR DESCRIPTION
**Pull request by maheshtamarakar**
## Purpose

The purpose of this PR is to fix #1152 

## Goals

Open IRC blogs in new tabs

## Approach

Added target="_blank"

### Screenshots

N/A
  
### Preview Link

https://pr-1153-sef-site.surge.sh/irc.html

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Related PRs
 
N/A

## Test environment
 
N/A

## Learning

N/A

### CODE REVIEW

**Comment by Piumal1999** [comment]
Well done @maheshtamarakar. Could you please update the pull request description, and mark the issue as 'ready for review'?

File: irc.html [hunk 1]
```diff
@@ -492,14 +492,14 @@ <h5 class="title">
         </thead>
         <tbody>
         {{#data}}
-          <tr onclick="window.location='{{linkedin}}';">
+          <tr>
             <th scope="row">
                 <img src="https://res.cloudinary.com/dsxobn1ln/image/upload/c_scale,h_48,w_48/{{image}}" class="avatar">
             </th>
-                <td>{{name}}</td>
+                <td ><a target="_blank" href="{{linkedin}}">{{name}}</a></td>
```
Comment (Piumal1999) [inline irc.html]:
```suggestion
                <td><a target="_blank" href="{{linkedin}}">{{name}}</a></td>
```
Remove the extra space inside the 'td' tag

Comment (maheshtamarakar) [reply irc.html]: ok

**Comment by maheshtamarakar** [comment]
> Well done @maheshtamarakar. Could you please update the pull request description, and mark the issue as 'ready for review'?

done

**Review by Piumal1999 — APPROVED** [review]
Good job! @maheshtamarakar

**Review by akeel230 — APPROVED** [review]
@maheshtamarakar Good Job mate.

**Pull request merged**